scripts/update-usbkey.0.esp.sh

```diff
@@ -199,7 +199,7 @@ if [[ -z "$old_boot_ver" ]]; then
 	exit 0
 fi
 
-if cmp $mountpoint/$loader_path $contents/$loader_path 2>/dev/null; then
+if cmp $mountpoint/$loader_path $contents/$loader_path >/dev/null; then
```

Comment:
Could also use the `-s` (silent) option, but this is obvious without knowing `cmp` intimately.